src/backend/pipeline/modules/index/index.py

```diff
@@ -10,4 +11,4 @@ class Index(ParallelModule):
 
     def __init__(self, data: Data, input_data: Any):
         """Initializes the index module and its runnables."""
-        super().__init__(data, [NDVI], name="Index", module_type=Modules.INDEX)
+        super().__init__(data, [Nutrient], name="Index", module_type=Modules.INDEX)
```

Comment:
Can you parametrize this in `config.py`?

Reply:
I made a workaround for this. The only caveat I can see at the moment is that Parallel Modules can only be done before or after Modules. This should be fine as Parallel Modules are used for indicies.

Reply:
This looks good, I would just move the `self.parallel_modules` into the dictionary for module initialization. The module type is `dict[Module, any]` and you can attach the `runnables` to the `any`. In any case, it's alright with me, you can resolve!! :D